### PR TITLE
Use injector instead of CD-ROM

### LINKED ISSUE #141: ipconfig takes too long
**Describe the bug**
Without setup internet connection ipconfig command on guest takes too long to execute (4 minutes). We should limit this to ~1 minute.

### LINKED ISSUE #182: Fix race conditions related to CD-ROM insertion
Right now we do create a CD-ROM with a startup script and the malware sample before each analysis. This special CD is inserted to a live VM and we rely on it being loaded very fast (otherwise a race condition will occur, this is especially visible on Windows 10).

https://github.com/CERT-Polska/drakvuf-sandbox/blob/a011bf2b530ab936d94865ce720acf9e90e6bbd5/drakrun/drakrun/run.py#L39-L41

This could be replaced by implementing a special new mode in injector, which will simply place the file on desktop using VMI.
https://github.com/tklengyel/drakvuf/pull/916

### PR DESCRIPTION
**Pull request by icedevml**
resolve #182
resolve #141

### CODE REVIEW

**Comment by icedevml** [comment]
TODO `ipconfig /renew` before sample is started

**Comment by icedevml** [comment]
TODO also support custom run command (e.g. rundll sth sth)

**Comment by icedevml** [comment]
TODO update drakvuf bundle URL in conftest

**Comment by icedevml** [comment]
required feature on the DRAKVUF part https://github.com/tklengyel/drakvuf/pull/916

**Comment by icedevml** [comment]
TODO update drakvuf submodule to master

File: drakrun/drakrun/main.py [hunk 1]
```diff
@@ -372,6 +353,38 @@ def process(self):
...
+                                "-r", kernel_profile,
+                                "-i", inject_pid,
+                                "-k", kpgd,
+                                "-m", "writefile",
+                                "-e", f"%USERPROFILE%\\Desktop\\{file_name}",
+                                "-B", os.path.join(workdir, file_name)]
+
+                self.log.info("Running injector...")
+                injector = subprocess.Popen(injector_cmd, stdout=subprocess.PIPE)
+                outs, errs = injector.communicate(b"", 20)
```
Comment (BonusPlay) [inline drakrun/drakrun/main.py:368]: In entire codebase we're using `subprocess.check_output`, this should do same?

File: drakrun/drakrun/main.py [hunk 2]
```diff
@@ -372,6 +353,38 @@ def process(self):
...
+
+                if injector.returncode != 0:
+                    raise subprocess.CalledProcessError(injector.returncode, injector_cmd)
+
+                injected_fn = json.loads(outs)['ProcessName']
+                net_enable = int(self.config.config['drakrun'].get('net_enable', '0'))
```
Comment (BonusPlay) [inline drakrun/drakrun/main.py:374]: `self.config.config['drakrun']` ugly :donotwant:

File: drakrun/drakrun/main.py [hunk 3]
```diff
@@ -372,6 +353,38 @@ def process(self):
...
+
+                if "%f" not in start_command:
+                    self.log.warning("No file name in start command")
+
+                cwd = '\\'.join(injected_fn.split("\\")[:-1])
+                start_command = f"cd {cwd} & " + start_command.replace("%f", injected_fn)
```
Comment (BonusPlay) [inline drakrun/drakrun/main.py:380]: You sure we don't need shelx on `cwd`?

File: drakrun/drakrun/main.py (hunk 1 above)
Comment (icedevml) [reply drakrun/drakrun/main.py:368]: `Popen` gives more control over what you do and we do use `Popen` to launch DRAKVUF just a few lines below this

we do use `check_output` mostly for the commands executed in draksetup

File: drakrun/drakrun/main.py (hunk 2 above)
Comment (icedevml) [reply drakrun/drakrun/main.py:374]: This is already used this way in multiple places within this file. I agree this needs refactor but this is out of scope of this PR

File: drakrun/drakrun/main.py (hunk 3 above)
Comment (icedevml) [reply drakrun/drakrun/main.py:380]: fixed

**Pull request merged**